### PR TITLE
Sidecar version update

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Update sidecar versions

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243|

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Installed CSI Unity Driver and created test pods.
sidecars update 1
sidecars update 2
sidecars update 3

### CODE REVIEW

**Review by karthikk92 — DISMISSED** [review]

File: helm/csi-unity/templates/_helpers.tpl [hunk 1]
```diff
@@ -20,7 +20,7 @@ Return the appropriate sidecar images based on k8s version
 {{- define "csi-unity.snapshotterImage" -}}
   {{- if eq .Capabilities.KubeVersion.Major "1" }}
     {{- if and (ge (trimSuffix "+" .Capabilities.KubeVersion.Minor) "21") (le (trimSuffix "+" .Capabilities.KubeVersion.Minor) "24") -}}
-      {{- print "k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1" -}}
+      {{- print "k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.1" -}}
```
Comment (rensyct) [inline helm/csi-unity/templates/_helpers.tpl:23]: Please revert this change

Comment (panigs7) [reply helm/csi-unity/templates/_helpers.tpl]: Done

[the remark this answers is not in the document]
Comment (panigs7) [reply helm/csi-unity/templates/_helpers.tpl]: Done

**Review by rensyct — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]
